plat/intel/soc/stratix10/include/platform_private.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019, Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __PLATFORM_PRIVATE_H__
+#define __PLATFORM_PRIVATE_H__
+#include <bl_common.h>
+
+/*******************************************************************************
+ * Function and variable prototypes
+ ******************************************************************************/
+void plat_configure_mmu_el3(unsigned long total_base,
+			unsigned long total_size,
+			unsigned long ro_start,
+			unsigned long ro_limit,
+			unsigned long coh_start,
+			unsigned long coh_limit);
+
+
+void plat_configure_mmu_el1(unsigned long total_base,
```

Comment:
Is this function ever going to be used? If you don't plan to have a regular BL2 at S-EL1 you should remove it. That would let you simplify the code in ``` plat/intel/soc/stratix10/aarch64/platform_common.c``` by removing the macro that generates the functions (you only need the EL3 version).

Reply:
We will be using this for bl31 stage, sharing the code. I'll be upstreaming bl31 as soon as it is fully verified.

Reply:
Ok